build-system/pr-check/utils.js

```diff
@@ -251,6 +253,18 @@ function storeBuildToWorkspace_(containerDirectory) {
         `/tmp/workspace/builds/${containerDirectory}/${outputDir}`
       );
     }
+    // Bento components are compiled inside the extension source file.
+    for (const componentFile of globby.sync('extensions/*/?.?/dist/*.js')) {
+      fs.ensureDirSync(
+        `/tmp/workspace/builds/${containerDirectory}/${path.dirname(
+          componentFile
+        )}`
+      );
+      fs.moveSync(
+        componentFile,
+        `/tmp/workspace/builds/${containerDirectory}/${componentFile}`
+      );
+    }
```

Comment:
Same question / suggestion about this block.

Reply:
Moved to #35414